docs/SQL/development/target-models/write-options.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Write Options
+id: write-options
+description: Write Options of Target Models
+sidebar_position: 6
+tags:
+  - concept
+  - model
+  - write options
+  - SQL
+  - scd2
+---
+
+You can use Write Modes such as Overwrite, Append, and Merge to materialize your data. The Write Options tab within your Target Model Gem guides you through building different merge approaches, such as historical data changes with SCD 2, which use incremental materialization strategy.
+
+Target Models are considered incremental models, which use a materialization strategy to update your data warehouse tables by loading and transforming just the changed data since the last run. Incremental models avoid processing your entire Dataset, which allows them to update only the new rows. This greatly reduces the time and resources you need to dedicate for your data transformations.
+
+## Write Modes
+
+You can overwrite the target data entirely, append to existing target data, or merge to address slowly changing dimension scenarios.
+
+- To select write modes, you must set the Target Model [Type & Format](type-and-format.md) to **Table**.
+
+Once you've selected your write mode, you'll see in the Code view that the table is stored as a `"materialized": "incremental",` table, with `"incremental_strategy:` set to whichever write mode and merge approach you choose.
```

Comment:
nit - can we remove the comma in the end of code section for `"materialized": "incremental",`

Reply:
Done.